formats/hocon/src/test/kotlin/kotlinx/serialization/hocon/HoconRootObjectsTest.kt

```diff
@@ -78,4 +77,14 @@ class HoconRootMapTest {
         // com.typesafe:config lib parse empty config as empty object
         ConfigFactory.parseString(emptyConfig)
     }
+
+    @Test
+    fun testEncodeList() {
+
```

Comment:
These ones probably have to do something :)

Reply:
Ooops! Forgot to remove these declarations